LAD-AMA-Common/metrics_ext_utils/metrics_ext_handler.py

```diff
@@ -352,8 +356,10 @@ def start_metrics(is_lad):
     if metrics_utils.is_systemd():
         service_restart_status = os.system("sudo systemctl restart {0}".format(metrics_service_name))
         if service_restart_status != 0:
-            log_messages += "Unable to start {0}. Failed to start ME service.".format(metrics_service_name)
-            return False, log_messages
+            service_restart_status = os.system("systemctl restart {0}".format(metrics_service_name))
+            if service_restart_status != 0:
+                log_messages += "Unable to start {0}. Failed to start ME service.".format(metrics_service_name)
```

Comment:
Looks good, can we append the error message to say "Unable to start ......" + ". Check if the system has some custom hardening or policies preventing systemctl command"